### ISSUE TITLE
Tiny border around windows

### ISSUE THREAD

**Issue by AnwarShah** [vinceliuice/Qogir-theme#139]
There is a tiny border around windows in Cinnamon desktop environment. This is annoying. Nemo has it, also terminal application has it.

How do I remove this? 

Nemo
![Border around nemo](https://user-images.githubusercontent.com/1885909/85674952-1a1c8a00-b6e7-11ea-803a-0585cc7326d4.png)

Gnome terminal
![Border around gnome terminal](https://user-images.githubusercontent.com/1885909/85675022-2c96c380-b6e7-11ea-8d34-8156b7d075ce.png)

**Comment by Sojiro84** [comment]
Why would you want it gone? If I remember correctly, in the old versions there was no border and that was frustrating me since having a border makes it clear where a window ends. Windows with no borders look just weird.

But if you want to change it, I think you need to look in the metacity.xml files.

**Comment by AnwarShah** [comment]
> Why would you want it gone? If I remember correctly, in the old versions there was no border and that was frustrating me since having a border makes it clear where a window ends. Windows with no borders look just weird.

Tiny 1 pixel white border is weirder imo. Also this is I think a bug because it's not present in Qogir dark. Only in normal. Although the dark titlebar is same in both variant.

**Comment by Sojiro84** [comment]
> > Why would you want it gone? If I remember correctly, in the old versions there was no border and that was frustrating me since having a border makes it clear where a window ends. Windows with no borders look just weird.
> 
> Tiny 1 pixel white border is weirder imo. Also this is I think a bug because it's not present in Qogir dark. Only in normal. Although the dark titlebar is same in both variant.

The dark version also has a border. It is almost black but it is there. I am using the dark version with the color of the border changed to a different color.

**Comment by AnwarShah** [comment]
> The dark version also has a border. It is almost black but it is there. I am using the dark version with the color of the border changed to a different color.

yeah, I know there is a border in dark version too. But that blends with surrounding color. Same should be in light version. No?

**Comment by cosmin-craciun** [comment]
I am also interested in removing that white border as I do not see a need for it. Have you managed to fix this?

**Comment by morsik** [comment]
FYI guys, I don't have any border in regular Qogir nor Qogir-dark on Gnome Shell, and border is not present on official screenshots of this theme so it looks like bug.

Also… 

> since having a border makes it clear where a window ends

@Sojiro84 true if you are not using shadows! If you have shadows, then no border looks a lot nicer with shadow in that case ;)

**Comment by Sojiro84** [comment]
> FYI guys, I don't have any border in regular Qogir nor Qogir-dark on Gnome Shell, and border is not present on official screenshots of this theme so it looks like bug.
> 
> Also…
> 
> > since having a border makes it clear where a window ends
> 
> @Sojiro84 true if you are not using shadows! If you have shadows (which this theme has, and I like that), then no border looks a lot nicer with shadow in that case ;)

The shadow on Cinnamon is barely visible, it's a tiny drop shadow. But this theme does have a border, but it's, last I checked, a black 1px border. That is why I edited on my end to a blue-ish color. No drop shadows + a lighter color looks good.

But I have since switched over to KDE Plasma for my theming needs (and as a bonus better performance on the DE as a whole).
